websites/N/Nekopoi/dist/metadata.json

```diff
@@ -0,0 +1,19 @@
+{
+    "$schema": "https://schemas.premid.app/metadata/1.3",
+    "author": {
+      "name": "Aizuu",
+      "id": "271576733168173057"
+    },
+    "service": "NekoPoi",
+    "description": {
+      "en": "Nekopoi is indonesian hentai and jav streaming site",
+      "id": "Nekopoi adalah tempat mencari hentai dan jav bersubtitle indonesia"
+    },
+    "url": "nekopoi.care",
+    "version": "1.0.0",
+    "logo": "https://i.imgur.com/u08RGtt.png",
+    "thumbnail": "https://i.imgur.com/GrEgIWx.png",
```

Comment:
different thumbnail with hidden nsfw things thanks :)

Reply:
omg ,,, im so sorry i will change this one

Reply:
lmao how did I not notice that xDDD